### PR TITLE
fix panic when stopping MySQLAnalyzer before its start

### PR DESCRIPTION
**Pull request by ikarishinjieva**
`MySQLAnalyzer.Stop()` is panic before invoding `MySQLAnalyzer.Start()`

### CODE REVIEW

File: qan/analyzer/mysql/mysql.go [hunk 1]
```diff
@@ -178,7 +178,11 @@ func (m *MySQLAnalyzer) Stop() error {
 	// instance are not affected.
 	m.mrms.Remove(m.protoInstance.UUID, restartChan)
 
-	return a.Stop()
+	if nil != a {
```
Comment (percona-csalguero) [inline qan/analyzer/mysql/mysql.go]: I think this should be moved to the first line after the function definition as
```go
func (m *MySQLAnalyzer) Stop() error {
    if m.analyzer == nil {
        return nil
    }
   .
   . 
   .
```

Comment (ikarishinjieva) [reply qan/analyzer/mysql/mysql.go]: Updated. Thanks.

**Review by arvenil — APPROVED** [review]

**Comment by arvenil** [comment]
Thanks!

**Comment by arvenil** [comment]
@ikarishinjieva Thank you for your contribution!

**Pull request merged**